keyboards/ergodash/keymaps/default/keymap.c

```diff
@@ -0,0 +1,137 @@
+#include "ergodash.h"
+#include "action_layer.h"
+#include "eeconfig.h"
+
+extern keymap_config_t keymap_config;
+
+#define _QWERTY 0
+#define _LOWER 1
+#define _RAISE 2
+#define _ADJUST 16
+
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
+  LOWER,
+  RAISE,
+  ADJUST,
+};
+
+// Fillers to make layering more clear
+#define KC_ KC_TRNS
+#define _______ KC_TRNS
+#define XXXXXXX KC_NO
+#define KC_AJST ADJUST
+#define KC_LOWR LOWER
+#define KC_RASE RAISE
+#define KC_JPN LALT(KC_GRV)
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+/* QWERTY
+ */
+  [_QWERTY] = KC_KEYMAP(
```

Comment:
I hate to ask, but we've had a number of problems (especially with new users) with the "KC" layouts.  The default keymap shouldn't use them.  It's fine to have a secondary that does. 

Also, if these can be changed to `LAYOUT_ortho_5x12` instead?